iphone/Maps/Categories/UIView+AddSeparator.swift

```diff
@@ -0,0 +1,17 @@
+extension UIView {
```

Comment:
What are the benefits of introducing an extension that is used only in one place? Wouldn't it be better to localize this code only in the class that uses it?

Reply:
This extension was used 2nd time here (previous one in PP screen), so i made it non-private.